web-local/src/lib/components/workspace/explore/metrics-container/MeasuresContainer.svelte

```diff
@@ -0,0 +1,139 @@
+<script lang="ts">
+  import {
+    useRuntimeServiceMetricsViewTotals,
+    V1MetricsViewTotalsResponse,
+  } from "@rilldata/web-common/runtime-client";
+  import { runtimeStore } from "@rilldata/web-local/lib/application-state-stores/application-store";
+  import { useMetaQuery } from "@rilldata/web-local/lib/svelte-query/dashboards";
+  import { EntityStatus } from "@rilldata/web-common/lib/entity";
+  import type { UseQueryStoreResult } from "@sveltestack/svelte-query";
+  import {
+    MetricsExplorerEntity,
+    metricsExplorerStore,
+  } from "../../../../application-state-stores/explorer-stores";
+  import MeasureBigNumber from "./MeasureBigNumber.svelte";
+
+  export let metricViewName;
+  export let metricsContainerHeight;
+
+  const MEASURE_HEIGHT = 60;
+  const MEASURE_HEIGHT_MULTILINE = 80;
+  const MEASURE_WIDTH = 175;
+  const MARGIN_TOP = 20;
+  const CHARACTER_LIMIT_FOR_WRAPPING = 26;
+
+  let metricsExplorer: MetricsExplorerEntity;
+  $: metricsExplorer = $metricsExplorerStore.entities[metricViewName];
+
+  $: instanceId = $runtimeStore.instanceId;
+
+  // query the `/meta` endpoint to get the measures and the default time grain
+  $: metaQuery = useMetaQuery(instanceId, metricViewName);
+
+  $: selectedMeasureNames = metricsExplorer?.selectedMeasureNames;
+
+  function getMeasureHeightsForColumn(measuresHeight, numColumns) {
+    const recalculatedHeights = [...measuresHeight];
+    for (let i = 0; i < measuresHeight.length; i = i + numColumns) {
+      const row = measuresHeight.slice(i, i + numColumns);
+      if (row.indexOf(MEASURE_HEIGHT_MULTILINE) != -1) {
+        for (let j = i; j < i + numColumns && j < measuresHeight.length; j++) {
+          recalculatedHeights[j] = MEASURE_HEIGHT_MULTILINE;
+        }
+      }
+    }
+    return recalculatedHeights;
+  }
+
+  let totalsQuery: UseQueryStoreResult<V1MetricsViewTotalsResponse, Error>;
+  $: numColumns = 1;
+  let measureGridHeights = [];
+
+  $: if (
+    metricsExplorer &&
+    metaQuery &&
+    $metaQuery.isSuccess &&
+    !$metaQuery.isRefetching
+  ) {
+    let totalsQueryParams = {
+      measureNames: selectedMeasureNames,
+      filter: metricsExplorer?.filters,
+    };
+
+    totalsQuery = useRuntimeServiceMetricsViewTotals(
+      instanceId,
+      metricViewName,
+      totalsQueryParams
+    );
+
+    const measures = $metaQuery.data?.measures;
+
+    let measuresHeight = measures.map((measure) => {
+      const label = measure?.label || measure?.expression;
+      if (label.length > CHARACTER_LIMIT_FOR_WRAPPING)
```

Comment:
does this mean this initialization is font-specific? 

I don't have a problem with this methodology for now, but a more robust method would (1) measure the DOM elements of these containers (2) determine the largest height for the row group and (3) apply that height to the other rows in the group.

This might mean that a reflow is needed, but if the reflow is done reactively as well, this should be fairly straightforward.